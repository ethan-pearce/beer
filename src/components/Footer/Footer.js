import './Footer.css'

const Footer = () => {
return (
        <footer className={'footer'}>
            <p>Â© Beer</p>
        </footer>
    );
}

export default Footer;