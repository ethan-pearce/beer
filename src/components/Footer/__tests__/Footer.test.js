import React from 'react'
import { render } from '@testing-library/react';
import Footer from '../Footer'

describe('Footer Component', () => {
    it('should render', () => {
        const { container } = render(<Footer />)

        expect(container.getElementsByClassName('footer').length).toBe(1)
        expect(container).toMatchSnapshot()
    })
})