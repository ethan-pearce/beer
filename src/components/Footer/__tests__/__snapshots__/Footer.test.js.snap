// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render 1`] = `
<div>
  <footer
    class="footer"
  >
    <p>
      Â© Beer
    </p>
  </footer>
</div>
`;
